docs/.vuepress/public/web_help/docs/zowe.html

```diff
@@ -15,21 +15,28 @@ <h2><a href="zowe.html" name="zowe">zowe</a></h2>
 <h4>Groups</h4>
 
 <ul>
-<li><a href="zowe_cics.html">cics</a> - Interact with IBM CICS programs and transactions.</li>
-<li><a href="zowe_config.html">config</a> - Manage configuration and overrides               </li>
-<li><a href="zowe_db2.html">db2</a> - Interact with IBM Db2 for z/OS                   </li>
-<li><a href="zowe_ims.html">ims</a> - Interact with IBM IMS programs and transactions. </li>
-<li><a href="zowe_mq.html">mq</a> - Interact with IBM MQ for z/OS.                   </li>
-<li><a href="zowe_plugins.html">plugins</a> - Install and manage plug-ins                      </li>
-<li><a href="zowe_profiles.html">profiles</a> - Create and manage configuration profiles         </li>
-<li><a href="zowe_provisioning.html">provisioning | pv</a> - Perform z/OSMF provisioning tasks                </li>
-<li><a href="zowe_zos-console.html">zos-console | console</a> - Issue z/OS console commands and collect responses</li>
-<li><a href="zowe_zos-files.html">zos-files | files</a> - Manage z/OS data sets                            </li>
-<li><a href="zowe_zos-jobs.html">zos-jobs | jobs</a> - Manage z/OS jobs                                 </li>
-<li><a href="zowe_zos-tso.html">zos-tso | tso</a> - Interact with TSO                                </li>
-<li><a href="zowe_zos-uss.html">zos-uss | uss</a> - Issue z/OS USS commands and receive responses    </li>
-<li><a href="zowe_zos-workflows.html">zos-workflows | wf</a> - Create and manage z/OSMF workflows               </li>
-<li><a href="zowe_zosmf.html">zosmf</a> - Interact with z/OSMF                               </li>
+<li><a href="zowe_cics.html">cics</a> - Interact with IBM CICS programs and              transactions.</li>
+<li><a href="zowe_config.html">config</a> - Manage configuration and overrides              </li>
+<li><a href="zowe_db2.html">db2</a> - Interact with IBM Db2 for z/OS                  </li>
+<li><a href="zowe_endevor.html">endevor</a> - CA Endevor SCM plug-in for listing Endevor       environment information, working with elements and packages located in specified Endevor instance.</li>
```

Comment:
The web help on zowe.org should not contain the Broadcom plugins (Endevor, FMP, OPS, Extended Files, Extended Jobs).